test/e2e/arktos/multi_tenancy/test_suites/kubectl/test_patch.yaml

```diff
@@ -0,0 +1,48 @@
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ Kubectl Patch Tests ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# This test suite verifies command "kubectl patch".
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+######################################################
+# test variables
+######################################################
+Variables:
+  test_tenant: random_8
+  test_ns: random_8
+
+###########################################################################################################
+# test setup
+###########################################################################################################
+Tests:
+  - BeforeTestMessage: Starting test setup ...
+    Command: ${kubectl} create tenant ${test_tenant}
+    OutputShouldContain: 
+    - "\ntenant/${test_tenant} created\n"
+
+  - Command: ${kubectl} create namespace ${test_ns} --tenant ${test_tenant}
+    OutputShouldBe: "namespace/${test_ns} created\n"
+
+  - Command:  ${kubectl} apply -f ${test_data_dir}/sample-pod.yaml -n ${test_ns} --tenant ${test_tenant} 
+    OutputShouldBe: "pod/sample-nginx-pod created\n"
+
+###########################################################################################################
+# test "kubectl patch"
+###########################################################################################################
+
+  - BeforeTestMessage: "Verifying \"kubectl patch\" ..."
+    Command: ${kubectl} get pod sample-nginx-pod -n ${test_ns} --tenant ${test_tenant} -o json | jq .spec.containers[0].image
+    OutputShouldBe: "\"nginx:1.7.9\"\n"
```

Comment:
will this 1.7.9 change over time?

Reply:
No. the value of 1.7.9 is from the test data file we set.